src/cm/profile.rs

```diff
@@ -9,6 +9,7 @@ use std::string::ToString;
 pub struct Profile {
     pub regex_list: StringList,
     pub cmd_list: StringList,
+    pub shell_list: StringList,
```

Comment:
Ok, we introduced this shell_list widget, but we never render it anywhere! How are we supposed to use it?

Reply:
You are right, the rendering was not implemented when you reviewed it.

There's an additional commit now which addresses your concern.
Please give it another chance.